project/Dependencies.scala

```diff
@@ -2,6 +2,7 @@ import sbt._
 
 object Dependencies {
   val akkaV = "2.4.1"
+  val akkaStreamV = "2.4.17"
```

Comment:
Will test out to see what works. 
Yes, 2.5 (which is latest akka streams) evicts the 2.4 akka code in spray.